Update kubernetes-walkthrough.md

alexandair opened the pull request:
Fix typos and the IP address to be the same in kubectl's output and in the image.

azurecla commented:
Hi __@brettsam__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (brettsam). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

alexandair committed:
Update kubernetes-walkthrough.md

Fix typos and the IP address to be the same in kubectl's output and in the image.
